tools/makenet.sh

```diff
@@ -3,6 +3,7 @@ EVE="$(cd "$(dirname "$0")" && pwd)/../"
 PATH="$EVE/build-tools/bin:$PATH"
 SOURCE="$(cd "$1" && pwd)"
 NET="$(cd "$(dirname "$2")" && pwd)/$(basename "$2")"
+IMG="$3"
```

Comment:
You have introduced a third argument which is mandatory. However, there is no information about it in the help message (line 9) and the checks on line 8 still considers only 2 arguments. Actually I think you could change the order, like this:

`Usage: <input dir> <image file name> <output tar image file>`

So we keep the output tar image file as the last argument, and please, consider use the entire file name as arguemnt, e.g., installer.img, verification.img, etc.